bot/database/__init__.py

```diff
@@ -0,0 +1,267 @@
+import typing as t
+from importlib import import_module
+
+import asyncpg
+from loguru import logger
+
+if t.TYPE_CHECKING:
+    from bot.core.bot import Bot
+
+
+class Singleton(type):
```

Comment:
Where is `type` from which You're inherting into singleton

Reply:
`type` is a builtin